llvm/lib/Transforms/InstCombine/InstCombineShifts.cpp

```diff
@@ -689,7 +690,8 @@ static Value *getShiftedValue(Value *V, unsigned NumBits, bool isLeftShift,
   IC.addToWorklist(I);
 
   switch (I->getOpcode()) {
-  default: llvm_unreachable("Inconsistency with CanEvaluateShifted");
```

Comment:
Don't reformat unrelated code

Reply:
I don't know why clang-format does that but I am trying to remove these changes.

Reply:
Use git clang-format rather than formatting the whole file. Or just `git add -p` / `git checkout -p` / etc to select which diffs you want.